server/models/Booking.js

```diff
@@ -1,4 +1,6 @@
-const { Schema } = require("mongoose");
+// const { Schema } = require("mongoose");
+const mongoose = require('mongoose');
+const { Schema } = require('mongoose');
```

Comment:
You can change these two lines to be one import statement like this:

```suggestion
const { Schema, model } = require('mongoose');
```